### ISSUE TITLE
Are you going to produce a time series for us data?

### ISSUE THREAD

**Issue by downFast** [CSSEGISandData/COVID-19#2482]
Hi,
thanks for the work.

Are you guys going to produce a time series for usa data? i know you have it for confirmed and deaths but I was wondering if you'd be producing a timeseries including data as per the us daily.

Thanks

**Comment by DavidViral** [comment]
Two time series tables are for the US confirmed cases and deaths, reported at the county level. They are named time_series_covid19_confirmed_US.csv, time_series_covid19_deaths_US.csv, respectively.

https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series

**Comment by xaviermerino** [comment]
Are there any for recovered?

**Comment by downFast** [comment]
> Two time series tables are for the US confirmed cases and deaths, reported at the county level. They are named time_series_covid19_confirmed_US.csv, time_series_covid19_deaths_US.csv, respectively.
> 
> https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series

yes I know, but the daily one have more data and I was wondering if this will be made into a timeseries too

**Comment by DavidViral** [comment]
I doubt it.
When they dropped the time series for the states for several weeks, I created a US state level time series based on their  daily reports.  When they started the new dedicated US time series I was not about to change my spreadsheet yet again to accommodate the new time series.  I aggregate the county info from the daily reports into state info so I don't have a time series by county and I don't care about that anyway.


    On Thursday, May 7, 2020, 2:58:15 AM EDT, downFast <notifications@github.com> wrote:  
 
 



Two time series tables are for the US confirmed cases and deaths, reported at the county level. They are named time_series_covid19_confirmed_US.csv, time_series_covid19_deaths_US.csv, respectively.

https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series


yes I know, but the daily one have more data and I was wondering if this will be made into a timeseries too

**Comment by downFast** [comment]
ye that's what i am doing too, I was just wondering if they'll do it, not sure why they don't

**Comment by mambon5** [comment]
I guess it's politics.. as usual

**Comment by downFast** [comment]
> I guess it's politics.. as usual

Well I am not sure what that would mean, It is not like they are hiding the data, it is presented on the daily csv.

**Comment by mambon5** [comment]
the recovered?? Then I will download. I am very furstrated that there are no recovered for UK or US. Idk what is going on with these anglosaxon countries.
